src/Raven.Server/NotificationCenter/Handlers/ServerNotificationCenterHandler.cs

```diff
@@ -33,7 +35,9 @@ public async Task Get()
                                     isValidFor(db, false) == false)
                                     continue; // not valid for this, skipping
                             }
-
+                            
+                            if (TrafficWatchManager.HasRegisteredClients)
+                                AddStringToHttpContext(writer.ToString(), TrafficWatchChangeType.Notifications);
```

Comment:
we need to write the `action.ToJson()` here since we reset the ctx in `writer.WriteToWebSocket()`

Reply:
Done